File: windows/src/GoalMaker.Core/Planning/PlanningLists.cs
namespace GoalMaker.Core.Planning;

/// <summary>Every list the planner shows, computed together from the same tasks and planning day.</summary>
public sealed record PlanningLists(
    DateOnly Today,
    TodaySections TodaySections,
    IReadOnlyList<TaskItem> Tomorrow,
    IReadOnlyList<TaskItem> Inbox,
    DaySummary Summary);

File: android/app/src/main/kotlin/com/goalmaker/app/application/planning/PlanningLists.kt
package com.goalmaker.app.application.planning

import java.time.LocalDate

/** Every list the planner shows, computed together from the same tasks and planning day. */
data class PlanningLists(
    val today: LocalDate,
    val todaySections: TodaySections,
    val tomorrow: List<TaskItem>,
    val inbox: List<TaskItem>,
    val summary: DaySummary,
)
